db/schema.rb

```diff
@@ -25,6 +25,15 @@
     t.datetime "updated_at"
   end
 
+  create_table "comments", force: :cascade do |t|
```

Comment:
Just edit the Schema file and remove the ones that were added to this. Check GitHub file changes on this branch and you can see what was added to the schema file there.